editor/editor_settings.cpp

```diff
@@ -1022,7 +1022,8 @@ void EditorSettings::setup_network() {
 	add_property_hint(PropertyInfo(Variant::INT, "network/debug/remote_port", PROPERTY_HINT_RANGE, "1,65535,1"));
 
 	// Editor SSL certificates override
-	_initial_set("network/ssl/editor_ssl_certificates", _SYSTEM_CERTS_PATH);
+	String certs = has_setting("network/ssl/editor_ssl_certificates") ? get("network/ssl/editor_ssl_certificates") : _SYSTEM_CERTS_PATH;
```

Comment:
Why is it done this way? Doesn't it mean that the "initial" (revert) value will always be the currently set one, so it's not revertable?

Reply:
No, it means that the value shown in the editor settings will be the last configured value if available, otherwise `_SYSTEM_CERTS_PATH` (which is an empty string).
The same is done above for debug address and port.

Reply:
But isn't it what `_initial_set` did already? It's designed so that it defines the default value, but doesn't override any custom value.

Reply:
I see, I'll fix that, probably `network/debug/remote_port` should use the same behaviour then. (`network/debug/remote_host` is a bit special instead)

Reply:
Well, `_initial_set` always calls `set` :( . I guess `setup_network` needs to be modified a bit.

Reply:
I guess `setup_network()` simply happens too late? `_initial_set` works fine for dozens of settings, but it's meant to be called before loading the editor settings. It sets the default value, and then it can be overridden by user values from `editor_settings-3.tres`.

Reply:
See `_load_defaults()`, which is called by the constructor.

Reply:
See added commit https://github.com/godotengine/godot/pull/29871/commits/fc411115f5ddb214c19b23499400f10266095e5f as a potential fix

Reply:
Looks good!